packages/mock-block-dock/src/mock-block-dock.tsx

```diff
@@ -7,18 +7,25 @@ import {
   LinkedAggregationDefinition,
 } from "@blockprotocol/graph";
 import { useGraphEmbedderService } from "@blockprotocol/graph/react";
-import { Box } from "@mui/material";
-import { ComponentType, FunctionComponent, useEffect, useRef } from "react";
+import {
+  ComponentType,
+  FunctionComponent,
+  lazy,
+  Suspense,
+  useEffect,
+  useRef,
+} from "react";
 
-import styles from "./assets/debug-view-styles.module.css";
 import { BlockRenderer } from "./block-renderer";
-import { DebugView } from "./debug-view";
-import { OffSwitch } from "./debug-view/icons";
 import { MockBlockDockProvider } from "./mock-block-dock-context";
 import { useDefaultState } from "./use-default-state";
 import { useMockBlockProps } from "./use-mock-block-props";
 import { useSendGraphValue } from "./use-send-graph-value";
 
+const MockBlockDockUi = lazy(async () => ({
+  default: (await import("./mock-block-dock-ui")).MockBlockDockUi,
```

Comment:
Not quite like in docs, but it should work too:
https://reactjs.org/docs/code-splitting.html#named-exports